low-level/files/opt/seagate/sspl/setup/sspl_provisioner_init

```diff
@@ -149,7 +149,7 @@ start_req_services(){
 if [ "$install_provisioner" == true ]; then
     setup_provisioner_prereq
 fi
-$SCRIPT_DIR/bin/sspl_post_install -p $PRODUCT_NAME -e DEV -c $rmq_cluster
+$SCRIPT_DIR/bin/sspl_setup post_install -p $PRODUCT_NAME -e DEV -c $rmq_cluster
```

Comment:
This script btw is getting replaced with sspl dev deploy script, so we should not use it actually. Plan to drop this in near future, after confirming everything we need is indeed in dev deploy script

Reply:
Sure.